klaviyo/public.py

```diff
@@ -63,8 +72,27 @@ def track(
         if ip_address:
             params['ip'] = ip_address
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.TRACK, query_string)
+        if method.lower() not in ['get','post']:
+
+            raise ValueError('method argument must be either "post" or "get"')
+
+        if method == 'post':
+
+            url = '{}/{}'.format(KlaviyoAPI.KLAVIYO_API_SERVER,self.TRACK)
```

Comment:
nit: double quotes and
`url = '{}/{}'.format(KlaviyoAPI.KLAVIYO_API_SERVER, self.TRACK)`